roles/endpoints/defaults/main.yml

```diff
@@ -238,6 +238,6 @@ endpoints:
       backend_api: 8043
       haproxy_api: 8042
     haproxy:
-      health_check: 'httpchk GET /healthcheck'
+      health_check: 'httpchk /'
```

Comment:
aodh-api has no api path like /healthcheck.

please reference to [aodh v2 api](http://docs.openstack.org/developer/aodh/webapi/v2.html)
Also only swift use "/healthcheck", other like heat, keystone all use "/"